src/js/components/DataTable/index.d.ts

```diff
@@ -117,6 +119,7 @@ export interface DataTableProps<TRowType = any> {
         expand?: (string | number)[];
         onExpand?: (expandedKeys: (string | number)[], datum: TRowType) => void;
       };
+  expandAriaLabel?: (row: TRowType, groupKey: any) => string;
```

Comment:
Will groupKey always be a string?
```suggestion
  expandAriaLabel?: (row: TRowType, groupKey: string) => string;
```